### DISCUSSION TITLE
Possible to save generated html files when using Jekyll remote theme?

### DISCUSSION THREAD

**Discussion by aronsommer** [community/community#23301] [Other Feature Feedback, Questions, & Ideas]
<p>My project is using this Jekyll theme as a remote theme:</p>
<p><a href="https://github.com/pmarsceill/just-the-docs" rel="nofollow noopener">https://github.com/pmarsceill/just-the-docs</a></p>
<p>Is there a way to save the generated html files?</p>
<p>I can not find a _site directory which includes the static site.</p>

**Comment by darrowby385** [comment, marked as answer]
<p>Hi <a class="mention" href="/aronsommer">@aronsommer</a>,</p>
<p>Rather than creating a _site directory in your GitHub repository, GitHub Pages runs the Jekyll build process separately, and stores the generated files elsewhere on GitHub’s servers.</p>
<p>It isn’t possible to directly download the built site, but you can create it yourself by cloning your repository and running a Jekyll build on your local machine. In order for your local environment to to resemble GitHub Pages as closely as possible (including support for the remote_theme configuration, among other things), you’ll need to install the github-pages gem, and add the following line to your site’s Gemfile:</p>
<pre><code>gem'github-pages',group::jekyll\_plugins
</code></pre>
<p>Further documentation on the GitHub Pages gem is available here:</p>
<p><a href="https://github.com/github/pages-gem#github-pages-ruby-gem" rel="nofollow noopener">https://github.com/github/pages-gem#github-pages-ruby-gem</a></p>
<p>I hope that helps!</p>
<p>Friday</p>

**Comment by aronsommer** [comment]
<p>Thank you <a class="mention" href="/darrowby385">@darrowby385</a></p>
<p>Now I understand :slight_smile:</p>
<p>I will try to build it locally.</p>
<p>Have a nice day!</p>
